content/v2.0-preview/_index.md

```diff
@@ -1,51 +1,53 @@
 ---
-title: "Overview"
+title: "Welcome"
 weight: -1
 cascade:
     version: "2.0-preview"
 ---
 
-{{< img src="/media/banner.png" alt="Crossplane Popsicle Truck" size="large" >}}
+Welcome to the Crossplane documentation. Crossplane is a control plane framework
+for platform engineering. 
 
-<br />
+Crossplane lets you build control planes to manage your cloud native software.
+It lets you design the APIs and abstractions that your users use to interact
+with your control planes.
```

Comment:
should we add just another snippet here to help people understand why users will interact with control planes, to accomplish what? (provision and manage their resources?)

Reply:
The tip box that explains what a control plane is already elaborates on that. Is that sufficient?